specification/marketplace/resource-manager/Microsoft.Marketplace/stable/2020-12-01/Marketplace.json

```diff
@@ -0,0 +1,897 @@
+      "type": "object",
+      "properties": {
+        "offerId": {
+          "description": "Gets offer id",
+          "type": "string"
+        },
+        "displayName": {
+          "description": "Gets offer display name",
+          "type": "string"
+        },
+        "publisherId": {
+          "description": "Gets or sets publisher id",
+          "type": "string"
+        },
+        "messageCode": {
+          "description": "Gets or sets the notification message id",
+          "type": "integer",
+          "format": "int64"
+        },
+        "icon": {
+          "description": "Gets or sets the icon url",
+          "type": "string"
+        },
+        "plans": {
+          "description": "Gets or sets removed plans notifications",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PlanNotificationDetails"
+          }
+        }
+      }
+    },
+    "PlanNotificationDetails": {
+      "description": "Plan notification details",
+      "type": "object",
+      "properties": {
+        "planId": {
+          "description": "Gets or sets the plan id",
+          "type": "string"
+        },
+        "planDisplayName": {
+          "description": "Gets or sets the plan display name",
+          "type": "string"
+        }
+      }
+    },
+    "AdminActionDetails": {
+      "description": "Admin action details payload",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AdminActionProperties",
+          "description": "Admin action properties data structure",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "AdminActionProperties": {
+      "description": "Admin action properties data structure",
+      "type": "object",
+      "properties": {
+        "publisherId": {
+          "description": "Gets or sets publisher Id",
+          "type": "string"
+        },
+        "adminAction": {
+          "description": "Gets or sets admin action",
+          "type": "string",
+          "enum": [
+            "Approved",
+            "Rejected"
+          ],
+          "x-ms-enum": {
+            "name": "adminAction",
+            "modelAsString": true
+          }
+        },
+        "existingRequestedPlans": {
+          "description": "Gets or sets existing plans ids ( of existing requests)",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "approvedPlans": {
+          "description": "Gets or sets Approved plans ids, empty in case of rejected",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "comment": {
+          "description": "Gets or sets admin comment",
+          "type": "string"
+        },
+        "user": {
+          "description": "Gets or sets admin details",
+          "type": "string"
+        }
+      }
+    },
+    "AdminGetApprovalRequestsResponse": {
```

Comment:
@pilor  same for this, does this need to be array as it is used under list call "/providers/Microsoft.Marketplace/privateStores/{privateStoreId}/offers/{offerId}/adminRequestApprovals"